src/Symfony/Component/Intl/ResourceBundle/LanguageBundle.php

```diff
@@ -81,7 +81,7 @@ public function getLanguageNames($displayLocale = null)
         try {
             return $this->getNames($displayLocale);
         } catch (MissingResourceException $e) {
-            return array();
+            return [];
```

Comment:
SF dont use this syntax (yet); so for consistency it's better to this all at once, or on a per-component base.